pkg/attestation/renderer/renderer.go

```diff
@@ -94,6 +94,16 @@ func NewAttestationRenderer(state *crafter.VersionedCraftingState, attClient pb.
 	return r, nil
 }
 
+// Render the in-toto statement without envelope nor validation so it can be used for example for policy evaluation
+func (ab *AttestationRenderer) RenderStatement(ctx context.Context) (*intoto.Statement, error) {
+	statement, err := ab.renderer.Statement(ctx)
```

Comment:
If I'm not wrong, this call is already evaluating policies, there's no need to evaluate them again.

Reply:
let me double check